src/main/scala/mesosphere/marathon/core/group/GroupManager.scala

```diff
@@ -8,10 +8,10 @@ import akka.stream.scaladsl.Source
 import mesosphere.marathon.core.instance.Instance
 import mesosphere.marathon.core.pod.PodDefinition
 import mesosphere.marathon.state.{ AppDefinition, Group, PathId, RootGroup, RunSpec, Timestamp }
-import mesosphere.marathon.upgrade.DeploymentPlan
 
 import scala.concurrent.Future
 import scala.collection.immutable.Seq
+import mesosphere.marathon.upgrade.DeploymentPlan
```

Comment:
because code style re-organizes imports alphabetically and before we annoy everyone reverting these changes before commit, it makes sense to have them in alphabetic order imo